app/views/shared/_application_information.html.erb

```diff
@@ -17,6 +17,12 @@
             <td class="govuk-table__cell"><strong>Address:</strong></td>
             <td class="govuk-table__cell"><%= @planning_application.site.full_address %></td>
           </tr>
+          <% unless @planning_application.site.postcode.blank? %>
```

Comment:
would we not want to make the site postcode required? It the site doesn't have a postcode the application shouldn't be valid?

Reply:
The postcode field does not currently validate presence. Adding validation is possibly the right thing to do but a) we'd need to check that that's correct (I think people can ask for planning permission on plots of land that don't have post codes? But would have to check that), and b) is outside the scope of this ticket (it would change the API, as post code would then be a required field)